src/disco/css/Addon.scss

```diff
@@ -6,9 +6,20 @@ $primary-text-color: #000;
 $secondary-text-color: #6a6a6a;
 
 .addon {
+  align-items: center;
   background: #fff;
+  display: flex;
+  flex-direction: row;
+  line-height: 1.5;
   margin-top: 20px;
 
+  &.theme {
```

Comment:
And these definitely don't all need to change right now since this is a bigger issue than these changes, but I'd rather see us add more specific classes than nest our classes. Maybe not even use `&` or at least not using it for a class, `&:before`, `&:hover`, etc might be fine.

I'd rather see `.addon--theme` or I guess `.Addon-theme` (or `.Addon--theme`?) as I proposed above.